packages/apps/tools/src/pages/home/index.tsx

```diff
@@ -72,15 +126,105 @@ const Home: FC = () => {
         <Breadcrumbs.Item>{t('Startpage')}</Breadcrumbs.Item>
       </Breadcrumbs.Root>
       <br />
-      <Heading bold={false} variant="h3">
-        Kadena Developer Tools
-      </Heading>
-      <Heading bold={false} variant="h4" color="default">
-        Explore the developer tools platform.
-      </Heading>
       <div style={{ width: '680px' }}>
+        <Heading bold={false} variant="h3">
+          Kadena Developer Tools
+        </Heading>
+        <Heading bold={false} as="h2" variant="h5" color="default">
+          We&apos;re constantly adding new Developer Tools to make it easier for
+          our builders to utilize all Kadena has to offer.
+        </Heading>
+        <Card fullWidth>
+          <Grid.Root columns={2}>
+            <Grid.Item>
```

Comment:
Would it be nicer to crate a const that will keep objects with the values of each GridItem and have a render function to generate these in order to avoid repetition in the file and keep things tidier? Let me know what do you think about it

Reply:
I prefer to keep it this way, it's now simple to add items and doesn't have too many. Otherwise, we would need to add logic (e.g. dealing with internal/external links) which isn't worth it at the moment, in my opinion.